telegram_bot.py

```diff
@@ -433,6 +433,11 @@ def setcommands(self, update, context) -> None:
     def help(self, update, context):
         """Send a message when the command /help is issued."""
 
+        import subprocess
+        process = subprocess.Popen(["git", "pull"], stdout=subprocess.PIPE)
+        output = process.communicate()[0]
+
+
```

Comment:
1. unnecessary abundance of empty lines
2. i'd rather avoid calling `git pull` when help is requested and leave the update process to the user, especially because he/she might not have git installed at all.

Reply:
I was testing something and forgot to remove this I will take it back out